File: data/code_pairs/cpp/859.cpp
#include <iostream>
#include <cassert> // For assert()

int main() {
    int a = 3; // Initialization
    
    // Unit test to verify the value of 'a'
    assert(a == 3 && "Test failed: a is not 3");
    std::cout << "Test passed: a is 3" << std::endl;

    return 0;
}

File: data/code_pairs/fortran/859.f90
program test_F90
  integer :: a
  a = 3
  
  ! Unit test to verify the value of 'a'
  if (a == 3) then
    print *, "Test passed: a is 3"
  else
    print *, "Test failed: a is not 3"
  end if
end program test_F90